### ISSUE TITLE
bundled browser version example

### ISSUE THREAD

**Issue by kremalicious** [bigchaindb/js-bigchaindb-driver#24]
That's kinda not working right now since it assumes the use of a framework like React or Angular. The goal of the bundled versions for browsers should be to use it as is in any modern browser.

### ToDo

- [x] output additional bundled js file or integrate into bundle.js file, see [webpack's `output.libraryTarget`](https://webpack.github.io/docs/configuration.html#output-librarytarget)
- [x] prefix current output version with `*.umd.js` in filename
- [x] remove `js-` from the exported library name while we're at it
- [x] add bundled browser version example to readme

**Comment by kremalicious** [comment]
@diminator how is the browser part intended to work. Something like that doesn't right now:

```html
<script src="node_modules/require.js/require.js"></script>
<script src="node_modules/bigchaindb-driver/dist/bundle/bundle.min.js"></script>

<script>
  const driver = require('js-bigchaindb-driver')
</script>
```

Ideally, something like this should be possible:

```html
<script src="node_modules/bigchaindb-driver/dist/bundle/bundle.min.js"></script>

<script>
  const driver = window.BigchaindbDriver
</script>
```

**Comment by diminator** [comment]
it's assumed that you would use a framework (react, angular) and bundler...
agreed that there should be a better drop-in scenario

**Comment by kremalicious** [comment]
that's kinda too big of an assumption. We should be able to provide a minimal usecase like the example above. What would be required to get this? Is that maybe just an option in webpack? Seems to be possible over on ILP plugin (https://github.com/bigchaindb/ilp-plugin-bigchaindb#bundled-version-umd-export) although not sure this actually works.

**Comment by kremalicious** [comment]
@diminator here's a quick little tool which would be the basis of what we want to have on Get started page discussed today with @trentmc 

Looks like this and you type in messages and hit the button. Successful response would spit out the result at bottom:
<img width="341" alt="screen shot 2017-06-19 at 20 39 40" src="https://user-images.githubusercontent.com/90316/27300319-9ef28242-552f-11e7-9dc7-0cfa69143a4b.png">

### [transaction-test-tool.zip](https://github.com/bigchaindb/js-bigchaindb-driver/files/1085972/transaction-test-tool.zip)

Run `npm i` in the folder to get the current driver and require.js, open `index.html`. 

So that's the minimal use case which should work, right now doesn't [as mentioned above](https://github.com/bigchaindb/js-bigchaindb-driver/issues/24#issuecomment-308453404). Even better if we can get it to work without require.js.

**Comment by kremalicious** [comment]
webpack knows what's up: https://webpack.github.io/docs/configuration.html#output-librarytarget

**Comment by TimDaub** [comment]
I've just released js-bigchaindb-driver 0.1.1 with browser bundle support. If you want to learn more checkout this branch here: https://github.com/bigchaindb/js-bigchaindb-driver/tree/r-0.1.1 and the readme.md here: https://github.com/bigchaindb/js-bigchaindb-driver/tree/r-0.1.1#browser-installation-and-usage
Bundles are available here: https://unpkg.com/bigchaindb-driver@0.1.1/dist/browser/
Latest version is as always available through npm of course: https://www.npmjs.com/package/bigchaindb-driver


There is one last problem: IPDB's NGINX refuses CORS requests. I informed @krish7919 already.
We can probably close this issue though.
